website/src/content/development/forms.mdx

```diff
@@ -610,43 +610,51 @@ Sometimes a form contains many different fields, resulting in the form component
 
 One way to improve that is to split the form component into multiple smaller components. As a rule of thumb you can create one separate component for each form field. For example a `<FormChannelNameField>`, a `<FormChannelKeyField>`, and so on.
 
-These components are simple wrapper components of the UI Kit field components but they contain and encapsulate all the logic related to the specific field. As a result, your form component can look like this:
+<Info>
+
+To use the `useField` hook the form must be initialized using the `<Formik>` component instead of using the `useFormik` hook, so that the React context is properly defined.
```

Comment:
Suggestion:
```suggestion
To use the `useField` hook the form must be wrapped using the `<Formik>` component instead of using the `useFormik` hook, so that the React context is properly defined.
```